src/Yesod/Auth/OAuth2/Auth0.hs

```diff
@@ -0,0 +1,53 @@
+{-# LANGUAGE OverloadedStrings #-}
+-- |
+-- OAuth2 plugin for <https://auth0.com>
+--
+-- * Authenticates against specific auth0 tenant
+-- * Uses Auth0 user id (a.k.a [sub](https://auth0.com/docs/api/authentication#get-user-info)) as credentials identifier
+--
+module Yesod.Auth.OAuth2.Auth0
+  (oauth2Auth0HostScopes
+  , defaultAuth0Scopes) where
```

Comment:
Do you mind applying the patch from Restyled to fix up the style here?

Look for the "you can manually apply..." section in [its output](https://restyled.io/gh/freckle/repos/yesod-auth-oauth2/jobs/2160459).